website/src/pages/use-cases/code-health.tsx

```diff
@@ -0,0 +1,366 @@
+        quote: `As an organization that values paying down tech debt, Workiva's Client Platform team started using Sourcegraph to help them efficiently propagate updates to dependencies across all of their repositories without any ongoing maintenance.`,
+        logoImage: '/external-logos/workiva-vector-logo.svg',
+        linkText: 'Read the case study',
+        link: '/case-studies/workiva-automates-large-scale-code-changes',
+        logoAlt: 'Workiva',
+    },
+]
+
+const blogListItems = [
+    {
+        title: 'How not to break a search engine or: What I learned about unglamorous engineering',
+        description: `When Sourcegraph switched to a new search query parser, you'd never know anything had changed.  This is an account of the rigorous testing that happened behind the scenes to ensure a seamless transition.`,
+        type: 'Blog post',
+        image: 'https://storage.googleapis.com/sourcegraph-assets/blog/how-not-to-break-a-search-engine-new.png',
+        href: '/blog/how-not-to-break-a-search-engine-unglamorous-engineering',
+    },
+    {
+        title: 'How we migrated entirely to CSS Modules using codemods and Sourcegraph Code Insights',
+        description: `Learn how Sourcegraph's Frontend Platform team overhauled our web application's design system and UI using codemods to automate a challenging global migration to CSS modules and Code Insights to track and communicate progress.`,
+        type: 'Blog post',
+        image: 'https://storage.googleapis.com/sourcegraph-assets/blog/code-insights-ga-blogs/migrating-to-css-modules.png',
+        href: '/blog/migrating-to-css-modules-with-codemods-and-code-insights',
+    },
+    {
+        title: 'How we added backend integration testing to our CI pipeline',
+        description: `Here's the story and the lessons learned from our work to remove all existing backend-related end-to-end tests and reliably run their corresponding unit and integration tests as part of our CI pipeline on all branches.`,
+        type: 'Blog post',
+        image: 'https://storage.googleapis.com/sourcegraph-assets/blog/backend-integration-testing/backend-integration-testing.png',
+        href: '/blog/integration-testing',
+    },
+]
+
+const UseCasePage: FunctionComponent<PageProps> = props => (
+    <Layout
+        location={props.location}
+        meta={{
+            title: 'Improve Code Health | Sourcegraph',
+            description:
+                'Tackle refactoring efforts and tech debt from legacy systems with automated pull requests across your entire codebase to boost code health.',
+            image: 'https://about.sourcegraph.com/sourcegraph-og.png',
+        }}
+        className="use-cases-page navbar-light"
+        hero={
+            <>
+                <div className={styles.useCaseHeader}>
+                    <div className="container pb-4">
+                        <div className="row">
+                            <div className="col-lg-7 mb-8 mt-7">
```

Comment:
`col-lg-7` => `col-md-7`? [Copy comment](https://github.com/sourcegraph/about/pull/5259/files#r861353249)

Reply:
Thanks, Becca! This one is handled in dd987d8.

Reply:
Please see: 3cb96fe